lighthouse-cli/test/smokehouse/frontends/smokehouse-bin.js

```diff
@@ -85,7 +88,11 @@ async function begin() {
   const jobs = argv.jobs !== undefined ? Number(argv.jobs) : undefined;
   const retries = argv.retries !== undefined ? Number(argv.retries) : undefined;
 
-  const lighthouseRunner = runners[/** @type {keyof typeof runners} */ (argv.runner)];
+  const runnerPath = runnerPaths[/** @type {keyof typeof runnerPaths} */ (argv.runner)];
+  if (argv.runner === 'bundle') {
+    console.log('\n✨ Be sure to have recently run this: yarn build-all');
```

Comment:
```suggestion
    console.log('\n✨ Be sure to have recently run this: yarn build-devtools');
```

Reply:
I was anticipating a time when we decide which bundle to test and we forget to update this message. It's still technically true :)